src/target/riscv/riscv.h

```diff
@@ -56,6 +56,13 @@ enum riscv_halt_reason {
 	RISCV_HALT_ERROR
 };
 
+enum riscv_isrmasking_mode {
+	RISCV_ISRMASK_AUTO,	/* not supported yet */
```

Comment:
Let's not include them in the enum at all then. If you want them listed as a future reminder, then comment them out.

Reply:
I left comment for the future.